odyssey/client/src/Profiles/UserProfile.svelte

```diff
@@ -42,20 +47,22 @@
 			info = response.info;
 			tiers = response.tiers;
 			posts = response.posts;
+			surveys = response.surveys;
 			subscribedTierId = response.tier_id;
 		}
 
 		posts.sort((a, b) => (a.date > b.date) ? 1 : -1);
 
-		result = {
+		follow_result = {
 			profile_name: user.username
 		};
 
-		is_following = await isFollowing(result);
+		is_following = await isFollowing(follow_result);
 
 		await loadMorePosts();
+		await loadMoreSurveys();
+		console.log(surveys);
```

Comment:
Please remove console logs

Reply:
Please review the new changes everything should be fine now

Reply:
Perfect! 👌